v1/openings/serializers/opening.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+from rest_framework import serializers
+
+from v1.contributors.models import Contributor
+from v1.contributors.serializers import ContributorSerializer
+from v1.teams.models.team import Team
+from ..models import Opening, Responsibility, Skill
+
+
+class OpeningSerializer(serializers.ModelSerializer):
+    reports_to = ContributorSerializer(many=True)
```

Comment:
generally, embedding entities in APIs is discouraged and leat to multiple problems in the future
its a sign of a bad design
entities should be as loosely coupled as possible